arm-multilib/CMakeLists.txt

```diff
@@ -163,7 +168,7 @@ foreach(lib_idx RANGE ${lib_count_dec})
 
             # FVP testing should default to off, so override any 
```

Comment:
Does this comment need to be revised now that both FVP and QEMU testing is optional?